CHANGELOG.md

```diff
@@ -37,6 +37,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * `jj config set` command allows simple config edits like
   `jj config set --repo user.email "somebody@example.com"`
 
+* Added `ui.log-word-wrap` option to wrap `jj log`/`obslog`/`op log` content
```

Comment:
(No action needed) In the commit description, you say:

> I'm not sure if patch content should be wrapped, so this option only applies to the template output for now.


I'm pretty sure the patch content in `--git` mode should *not* be wrapped by default as that would break copy-paste. For the default mode, it'd be interesting to try wrapping, since it's not fit for copy-pasting already.

Reply:
Yeah, I think I agree. For color-words diff, maybe better to handle wrapping by the diff function so it can indent wrapped lines.